src/seperator.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 The Druid Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! A Seperator widget.
+
+use druid::widget::prelude::*;
+use druid::{kurbo::Line, piet::StrokeStyle};
+use druid::{theme, Color, KeyOrValue};
+
+/// A seperator widget.
+pub struct Seperator {
+    size: KeyOrValue<f64>,
+    color: KeyOrValue<Color>,
+    orientation: Orientation,
+    stroke_style: StrokeStyle,
+}
+
+pub enum Orientation {
+    Vertical,
+    Horizontal,
+}
+
+impl Default for Seperator {
+    fn default() -> Self {
+        Seperator {
+            size: theme::BUTTON_BORDER_WIDTH.into(),
+            color: theme::BORDER_LIGHT.into(),
+            orientation: Orientation::Horizontal,
+            stroke_style: StrokeStyle::new(),
+        }
+    }
+}
+
+impl Seperator {
+    pub fn new() -> Self {
+        Self::default()
+    }
+    pub fn with_size(mut self, size: impl Into<KeyOrValue<f64>>) -> Self {
+        self.size = size.into();
+        self
+    }
+
+    pub fn set_size(&mut self, size: impl Into<KeyOrValue<f64>>) {
+        self.size = size.into();
+    }
+
+    pub fn with_color(mut self, color: impl Into<KeyOrValue<Color>>) -> Self {
+        self.color = color.into();
+        self
+    }
+
+    pub fn with_stroke_style(mut self, stroke_style: StrokeStyle) -> Self {
```

Comment:
`set_stroke_style` seems to be  missing.

Reply:
oh I forgot that. adding...